example/src/components/ChatboxTest.tsx

```diff
@@ -61,10 +64,37 @@ const ChatboxTest: FC<Props> = () => {
   }));
 
   const classes = useStyles();
+  const { data: currentMember } = hooks.useCurrentMember();
+  const memberId = currentMember?.get('id') as string;
+  // mutations to handle the mentions
+  const { mutate: patchMentionMutate } = useMutation<
+    ChatMention,
+    unknown,
+    { memberId: string; id: string; status: string }
+  >(MUTATION_KEYS.PATCH_MENTION);
+  const patchMentionFunction = (args: { id: string; status: string }): void =>
+    patchMentionMutate({ memberId, ...args });
+  const { mutate: deleteMentionMutate } = useMutation<
+    ChatMention,
+    unknown,
+    { memberId: string; mentionId: string }
+  >(MUTATION_KEYS.DELETE_MENTION);
+  const deleteMentionFunction = (mentionId: string): void =>
+    deleteMentionMutate({ memberId, mentionId });
+  const { mutate: clearAllMentionsMutate } = useMutation<
+    ChatMention[],
+    unknown,
+    { memberId: string }
+  >(MUTATION_KEYS.CLEAR_MENTIONS);
+  const clearAllMentionsFunction = (): void =>
+    clearAllMentionsMutate({ memberId });
 
   // adapt the width of the chatbox to simulate the width used on Graasp
-  const onChangePanelWidth = (_: unknown, newValue: number | number[]) => {
-    if (typeof newValue == 'number') {
+  const onChangePanelWidth = (
+    _: unknown,
+    newValue: number | number[],
```

Comment:
What happens if `newValue` is an array?

Reply:
In the current setup it can not be an array. This is because the component i use `Slider` can be used for ranges (2 values). So this is why i have it use the `number | number[]` type here. The else clause should take care of it in case an array was passed.